rasa/core/processor.py

```diff
@@ -560,55 +539,53 @@ def _get_action(
             action_name, self.domain, self.action_endpoint
         )
 
-    async def parse_message(
-        self, message: UserMessage, tracker: Optional[DialogueStateTracker] = None
+    def parse_message(
+        self, message: UserMessage, only_output_properties: bool = True
     ) -> Dict[Text, Any]:
-        """Interprete the passed message using the NLU interpreter.
+        """Interpret the passed message.
 
         Arguments:
             message: Message to handle
-            tracker: Dialogue context of the message
 
         Returns:
             Parsed data extracted from the message.
         """
-        # preprocess message if necessary
-        if self.message_preprocessor is not None:
-            text = self.message_preprocessor(message.text)
-        else:
-            text = message.text
-
-        # for testing - you can short-cut the NLU part with a message
-        # in the format /intent{"entity1": val1, "entity2": val2}
-        # parse_data is a dict of intent & entities
-        if text.startswith(INTENT_MESSAGE_PREFIX):
-            parse_data = await RegexInterpreter().parse(
-                text, message.message_id, tracker
-            )
-        else:
-            parse_data = await self.interpreter.parse(
-                text, message.message_id, tracker, metadata=message.metadata
-            )
+        results = self.graph_runner.run(
+            inputs={
+                PLACEHOLDER_MESSAGE: [message] if message else [],
+                PLACEHOLDER_TRACKER: DialogueStateTracker("no_sender", []),
+            },
+            targets=["output_provider"],
+        )
+        parsed_message, _, _ = results["output_provider"]
+        parse_data = {
+            TEXT: "",
+            INTENT: {INTENT_NAME_KEY: None, PREDICTED_CONFIDENCE_KEY: 0.0},
+            ENTITIES: [],
+        }
```

Comment:
can we get similar behavior by using an empty `Message` / or something like a `Message.empty()`? Ideally the structure of `Message` internals is only defined in one place